web/src/styles/variables.scss

```diff
@@ -6,6 +6,9 @@ $instagram-primary: #f8f8f84d;
 $text-color-primary: #000000;
 $text-color-secondary: #010311;
 $text-color-accent: #ffffff;
+$primary-accent: #FF5A00;
+$secondary-accent: #ff5a0008;
```

Comment:
This doesn't look like a valid colour?

Reply:
@eddiejaoude Color code with opacity will be in this format.

Reply:
Oh ok, thanks for clarifying 👍